base/version.jl

```diff
@@ -121,18 +129,6 @@ function VersionNumber(v::AbstractString)
     return VersionNumber(major, minor, patch, prerl::VerTuple, build::VerTuple)
 end
 
-parse(::Type{VersionNumber}, v::AbstractString) = VersionNumber(v)
-function tryparse(::Type{VersionNumber}, v::AbstractString)
-    try
-        return VersionNumber(v)
-    catch e
-        if isa(e, InterruptException)
```

Comment:
Wow, this really swallowed any exception except `InterruptException`...  :scream: